app/src/main/java/com/kickstarter/extensions/AnswersExt.kt

```diff
@@ -6,3 +6,7 @@ import com.crashlytics.android.answers.CustomEvent
 fun fabricLogCustomEvent(customEvent: String) {
```

Comment:
Should we kill this extension until we decide we're using Fabric officially?

Reply:
Killed!